src/System.Reflection.Metadata/src/System/Reflection/Blob.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I guess I'll write a script to update them each time I sync between Roslyn and CoreFx.